File: Jack.Presentation/Controllers/MVC/RoupasController.vb
Imports System.Web.Mvc

Namespace Controllers.MVC
    Public Class RoupasController
        Inherits Controller

        ' GET: Roupas
        Function Index() As ActionResult
            Dim RoupaBusiness As Business.Roupa = Nothing
            Dim RoupaRetorno As List(Of Model.Roupa) = Nothing

            Try

                RoupaBusiness = New Business.Roupa()
                'fake init
                RoupaRetorno = New List(Of Model.Roupa)
                RoupaRetorno.Add(New Model.Roupa)
                RoupaRetorno.Add(New Model.Roupa)
                RoupaRetorno.Add(New Model.Roupa)
                RoupaRetorno.Add(New Model.Roupa)
                RoupaRetorno.Add(New Model.Roupa)
                'fake end
                'RoupaRetorno = RoupaBusiness.LoadAll()

            Catch ex As Exception
                RoupaRetorno = New List(Of Model.Roupa)
            Finally
                RoupaBusiness = Nothing
            End Try

            Return View(RoupaRetorno)

        End Function
    End Class
End Namespace

File: Jack.Presentation.Web/Controllers/MVC/RoupasController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web.Mvc;
using Application = Jack.Application;
using Model = Jack.Model;


namespace Controllers.MVC
{
    public class RoupasController : Controller
	{

		// GET: Roupas
		public ActionResult Index()
		{
			Application.Roupa RoupaBusiness = null;
			List<Model.Roupa> RoupaRetorno = null;


            try
            {
                RoupaBusiness = new Application.Roupa();
                //fake init
                RoupaRetorno = new List<Model.Roupa>();
                RoupaRetorno.Add(new Model.Roupa());
                RoupaRetorno.Add(new Model.Roupa());
                RoupaRetorno.Add(new Model.Roupa());
                RoupaRetorno.Add(new Model.Roupa());
                RoupaRetorno.Add(new Model.Roupa());
                //fake end
                //RoupaRetorno = RoupaApplication.LoadAll()

            }
            catch (Exception ex)
            {
                RoupaRetorno = new List<Model.Roupa>();
                throw ex;
            }
            finally
            {
                RoupaBusiness = null;
            }

			return View(RoupaRetorno);

		}
	}
}
